### ISSUE TITLE
Item Display Accessibility Issues

### ISSUE THREAD

**Issue by Blackop778** [cerner/terra-clinical#378]
# Bug Report

## Description
The `terra-clinical-item-display` ItemComment text color does not have enough contrast with the `terra-list` selected background color, the `terra-table` selected background color, or the `terra-table` striped background color.

## Steps to Reproduce
1. [Visit the selectable test page](https://engineering.cerner.com/terra-clinical/#/tests/terra-clinical-item-collection/clinical-item-collection/item-collection-selectable)
2. Test color contrast using Axe in table mode (larger viewport)
3. Test color contrast using Axe in list mode (smaller viewport)
4. [Visit the table styles test page](https://engineering.cerner.com/terra-clinical/#/tests/terra-clinical-item-collection/clinical-item-collection/item-collection-table-styles)
5. Test color contrast using Axe in table mode (larger viewport)

### Selectable
<img width="133" alt="screen shot 2018-07-24 at 2 43 56 pm" src="https://user-images.githubusercontent.com/11130785/43162282-26acdb1e-8f50-11e8-90a3-f45bad7f62f1.png">

```
expected no accessibility violations but got:
	{
  "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
  "help": "Elements must have sufficient color contrast",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.0/color-contrast?application=axeAPI",
  "id": "color-contrast",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#e2f2fc",
            "contrastRatio": 3.94,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#75767d",
            "fontSize": "10.5pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.94 (foreground color: #75767d, background color: #e2f2fc, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<li tabindex=\"0\" id=\"item3\" class=\"List-module__list-item___1QAkf List-module__selected___rDzz4 List-module__is-selectable___38JEl\">",
              "target": [
                "#item3"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.94 (foreground color: #75767d, background color: #e2f2fc, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<div class=\"ItemDisplay__text___2ZB9-\">test comment</div>",
      "impact": "serious",
      "none": [],
      "target": [
        "#item3 > .ItemView__item-view___2qBlH.ItemView__one-column___346PH > .ItemView__body___1Uda5 > .ItemComment__item-comment___IfOdl.ItemDisplay__item-display___3gBY_ > .ItemDisplay__text___2ZB9-"
      ]
    }
  ],
  "tags": [
    "cat.color",
    "wcag2aa",
    "wcag143"
  ]
}
```

### Striped
<img width="118" alt="screen shot 2018-07-24 at 3 00 06 pm" src="https://user-images.githubusercontent.com/11130785/43162973-4b9ec96c-8f52-11e8-9e26-41d0c52dadf1.png">

```
expected no accessibility violations but got:
	{
  "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
  "help": "Elements must have sufficient color contrast",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.0/color-contrast?application=axeAPI",
  "id": "color-contrast",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#f4f4f4",
            "contrastRatio": 4.11,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#75767d",
            "fontSize": "10.5pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 4.11 (foreground color: #75767d, background color: #f4f4f4, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<tr aria-selected=\"false\" class=\"Table-module__row___3pCTk\">",
              "target": [
                ".Table-module__striped___1iYyS > tbody > tr:nth-child(2)"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 4.11 (foreground color: #75767d, background color: #f4f4f4, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<div class=\"ItemDisplay__text___2ZB9-\">test comment</div>",
      "impact": "serious",
      "none": [],
      "target": [
        ".Table-module__striped___1iYyS > tbody > tr:nth-child(2) > .ItemCollection__content-comment___1-Riz > .ItemComment__item-comment___IfOdl.ItemDisplay__item-display___3gBY_ > .ItemDisplay__text___2ZB9-"
      ]
    }
  ],
  "tags": [
    "cat.color",
    "wcag2aa",
    "wcag143"
  ]
}
```
## Expected Behavior
No accessibility errors will be encountered

## Possible Solution
Change one of the colors

**Comment by bjankord** [comment]
@neilpfeiffer can we get some guidance on colors to get this to meet recommended WCAG 2 AA contrast ratios.

**Comment by ShettyAkarsh** [comment]
@neilpfeiffer could you please provide the foreground color suggestion for the clinical item display ? current color hexa value of foreground color for clinical  item display is #75767d.

**Comment by neilpfeiffer** [comment]
Thanks @ShettyAkarsh - Both the icon and color we use for the “secondary” text treatment should be changed to an rgba so the opacity allows for the change in the  background color to “add” to the light grey color and darken to pass when visually sampled. I will look up what the color is that was chosen to replace hex 75767d.

**Comment by neilpfeiffer** [comment]
@ShettyAkarsh -- new color will be `rgba(28, 31, 33, 0.65)`
for the text color for "secondary" textStyle in `terra-clinical-item-display` 
plus the text color for the `ItemDisplay.Comment`.   

The icon color for `ItemDisplay.Comment` can stay at `rgba(117, 118, 125, 0.30)`.
